src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  ElementRef,
+  Injectable,
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints
+} from '../media-query';
+
+/* tslint:disable */
+const SKY_TABBABLE_SELECTOR = [
+  'a[href]',
+  'area[href]',
+  'input:not([disabled]):not([tabindex=\'-1\'])',
+  'button:not([disabled]):not([tabindex=\'-1\'])',
+  'select:not([disabled]):not([tabindex=\'-1\'])',
+  'textarea:not([disabled]):not([tabindex=\'-1\'])',
+  'iframe',
+  'object',
+  'embed',
+  '*[tabindex]:not([tabindex=\'-1\'])',
+  '*[contenteditable=true]'
+].join(', ');
+/* tslint:enable */
+
+@Injectable()
+export class SkyAdapterService {
```

Comment:
I think we might need a better name. I totally get that this is an adapter but I'm wondering if we will get confused down the road with what this is vs the other adapters we have. Thoughts?

Reply:
`SkyCoreAdapterService`?

Reply:
Maybe? Lol. I think that is better than what is currently here.

Reply:
Done.